js/tests/index.html

```diff
@@ -7,7 +7,7 @@
 
     <!-- jQuery -->
     <script src="../../docs/assets/js/vendor/jquery-slim.min.js"></script>
-    <script src="../../docs/assets/js/vendor/tether.min.js"></script>
+    <script src="../../docs/assets/js/vendor/popper.min.js"></script>
```

Comment:
I found the reason why...
I didn't take the ES5 version of Popper... Sorry @FezVrasta 😄